File: control-assurance-sp/src/types/GIAAAuditReportDirectorate.ts
import { IEntity } from "./Entity";
import { IUser } from "./User";

export interface IGIAAAuditReportDirectorate extends IEntity {
    GIAAAuditReportId?: number;
    DirectorateId?: number;

}

export class GIAAAuditReportDirectorate implements IGIAAAuditReportDirectorate {
    public ID = 0;
    public Title = null;
    public GIAAAuditReportId = null;
    public DirectorateId = null;
}

File: ControlAssuranceAPI/Models/GIAAAuditReportDirectorate.cs
using System;
using System.Collections.Generic;

namespace CAT.Models;

public partial class GIAAAuditReportDirectorate
{
    public int ID { get; set; }

    public string? Title { get; set; }

    public int? GIAAAuditReportId { get; set; }

    public int? DirectorateId { get; set; }

    public virtual Directorate? Directorate { get; set; }

    public virtual GIAAAuditReport? GIAAAuditReport { get; set; }
}
